.gitlab-ci.yml

```diff
@@ -747,23 +757,40 @@ publish-rustdoc:
     - rm -rf .git/config
     - git config user.email "devops-team@parity.io"
     - git config user.name "${GITHUB_USER}"
-    - git config remote.origin.url "https://${GITHUB_TOKEN}@github.com/paritytech/substrate.git"
+    - git config remote.origin.url "https://${GITHUB_TOKEN}@github.com/paritytech/${CI_PROJECT_NAME}.git"
     - git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
     - git fetch origin gh-pages
+    # Install `ejs` and generate index.html based on RUSTDOCS_DEPLOY_REFS
+    - yarn global add ejs
+    - "ejs ${INDEX_TPL} -i \"{\\\"deploy_refs\\\":\\\"${RUSTDOCS_DEPLOY_REFS}\\\",\\\"repo_name\\\":\\\"${CI_PROJECT_NAME}\\\",\\\"latest\\\":\\\"${LATEST}\\\"}\" > /tmp/index.html"
```

Comment:
This won't work surrounded with `" "` as it has a lot of the same brackets inside, please use `' ' `